packages/vite/src/node/ssr/ssrModuleLoader.ts

```diff
@@ -35,7 +36,8 @@ export async function ssrLoadModule(
   url: string,
   server: ViteDevServer,
   context: SSRContext = { global },
-  urlStack: string[] = []
+  urlStack: string[] = [],
+  config?: ResolvedConfig
```

Comment:
I think for the config, we can access via `server.config`, that way we can avoid the extra parameter.